React/Views/RCTModalHostView.m

```diff
@@ -257,4 +264,11 @@ - (UIInterfaceOrientationMask)supportedOrientationsMask
 }
 #endif
 
```

Comment:
We'll need to add this to support changing the presentation style after setting the delegate
```objective-c
// Method must be implemented, otherwise iOS defaults to 'automatic' (pageSheet on >= iOS 13.0)
- (UIModalPresentationStyle)adaptivePresentationStyleForPresentationController:(UIPresentationController *)controller traitCollection:(UITraitCollection *)traitCollection
{
  return self.presentationStyle;
}
```